src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,47 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerDao manufacturerDao =
+                (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
+        Manufacturer manufacturerAudi = new Manufacturer(null, "Nissan", "Japan");
+        Manufacturer manufacturerBmw = new Manufacturer(null, "Porsche", "Germany");
+        manufacturerAudi = manufacturerDao.create(manufacturerAudi);
+        manufacturerBmw = manufacturerDao.create(manufacturerBmw);
```

Comment:
See the task: Main should only contain invocation of Service methods, no Dao methods allowed in main.